ansible/roles/mixing/templates/remix.sh.j2

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+result=$(dash-cli getbalances)
+coinjoin_value=$(echo "$result" | jq '.mine.coinjoin')
+
+# Check if coinjoin_value is greater than 30
+if (( $(echo "$coinjoin_value > 30" | bc -l) )); then
```

Comment:
Let's add logging here so we can easily see when recent transactions were sent

Reply:
@ktechmidas I attempted this in https://github.com/dashpay/dash-network-deploy/pull/528/commits/44135d223ccae0853b3fdd6791770d1895ee8098